### PR TITLE
chore: Bump logger_conf

### PR DESCRIPTION
**Pull request by ghislainbourgeois**
Bump `logger_conf` to not pull `path_util`.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,12 +1,38 @@
 module github.com/omec-project/util_3gpp
 
-go 1.14
+go 1.21
+
+toolchain go1.21.6
```
Comment (gab-arrobo) [inline go.mod]: Why is this line required? Is the goal to enable `toolchain` across all repos?

**Review by gab-arrobo — COMMENTED** [review]
@ghislainbourgeois, both GHA actions failed:
- Build fails because of the Go version specified in the Action ([go-version: '1.14.4'](https://github.com/omec-project/util_3gpp/blob/main/.github/workflows/go.yml#L23)). I think it needs to be updated as follows: `go-version-file: 'go.mod'`
- Lint fails because `deadcode`, `structcheck` and `varcheck` are considered as deprecated

It would be good to refactor the Actions to be similar as the other repos. Also, it would be good to add a `Dependabot`

File: go.mod (hunk 1 above)
Comment (ghislainbourgeois) [reply go.mod]: I did not realize this was added, I removed it.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 Canonical Ltd.
```
Comment (gab-arrobo) [inline .github/workflows/main.yml]:
```suggestion
# Copyright 2024 Canonical Ltd.
```

**Review by gab-arrobo — COMMENTED** [review]
What about adding a dependabot?

**Review by gab-arrobo — APPROVED** [review]
+1 (Tested changes with E2E deployment of AiaB by using the `replace` directive for the SMF image)
